account_analytic_project/view/account_analytic_account.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <record id="account_analytic_account_form" model="ir.ui.view">
+            <field name="name">account.analytic.account.form.add_project_button</field>
+            <field name="model">account.analytic.account</field>
+            <field
+                name="inherit_id"
+                ref="analytic.view_account_analytic_account_form"
+            />
+            <field name="arch" type="xml">
+                <div name="buttons" position="inside">
+                    <button
+                        name="create_project_for_account"
+                        type="object"
+                        class="oe_stat_button"
+                        icon="fa-plus"
+                        string="Create project"
+                        attrs="{'invisible': [
+                            '|', ('project_ids', '!=', False),
+                            ('type', '=', 'view'),
+                        ]}"
+                    />
+                    <button
+                        name="view_projects_for_account"
+                        type="object"
+                        class="oe_stat_button"
+                        icon="fa-tasks"
+                        string="View projects"
+                        attrs="{'invisible': [
+                            '|', ('projects_ids', '=', False),
```

Comment:
Now I understand. I thought the issue was a missing 's' that I did not see missing. In reality the comment correctly was about a 's'  to many.